src/pybind/mgr/dashboard/frontend/src/app/shared/services/summary.service.ts

```diff
@@ -40,7 +42,46 @@ export class SummaryService {
     });
   }
 
+  /**
+   * Returns the current value of summaryData
+   *
+   * @returns
+   * @memberof SummaryService
+   */
   get() {
-    return this.http.get('api/summary');
+    return this.summaryDataSource.getValue();
+  }
+
+  /**
+   * Subscribes to the summaryData,
+   * which is updated at least once every 5 seconds
```

Comment:
I'm not sure if this comment brings any value. I would prefer instead a small comment after `this.refresh()` in line 39 that this will trigger all subscribers of `subscribe`.

Reply:
When the developer are calling this method, they usually are in another file, so they can't read the commends on the file. 
But if their IDE support JSDoc it will show this information to the developer and they don't need to search what it will happen after calling this.

Reply:
But could you than rephrase it? How about `Subscription of 'summaryData' will trigger 'call' every 5 seconds`

I'm not sure but `at least` seems to be not needed here as far as I can tell only the request done in the `timeout` will change the `summaryData`.

Reply:
Good to know - could you add that to the description? That's really helpful :+1:

Reply:
Good to know. Could you add that to the description as it is a really useful information :+1:

Reply:
Done.